Sources/TuistGraph/Graph/GraphOutputJSON.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import TSCBasic
+
+public struct GraphOutputJSON: Codable, Equatable {
```

Comment:
I like the name `GraphOutput` a bit more. I don't think the `JSON` suffix is necessary here.

Reply:
these are all public methods as well, so I think we should document the models and public properties.

Reply:
Sounds good.